joomla/joomla-cms
# [com_languages] installed view: Add missing tooltip in search

#### Summary of Changes

Simple PR to add a missing tootip in the com_languages, installed view search filters.
#### Testing Instructions

Code review, or:
1. Apply patch
2. Go to Extensions -> Languages -> Installed, check the tooltip in the search field.
